scripts/localk8s/uninstall-app.bat

```diff
@@ -0,0 +1,27 @@
+@ECHO OFF
+SET SCRIPTLOC=%~dp0
+
+:: Uninstall microservices
+helm uninstall kc-ui -n shipping
+helm uninstall order-command-ms -n shipping
+helm uninstall order-query-ms -n shipping
+helm uninstall spring-container-ms -n shipping
+helm uninstall voyages-ms -n shipping
+helm uninstall fleet-ms -n shipping
+
+:: Remove Kafka topics
+kubectl delete -f %SCRIPTLOC%\topics.yaml
+
+:: Delete Postgres secrets
+kubectl delete secret postgresql-url -n shipping
+kubectl delete secret postgresql-user -n shipping
+kubectl delete secret postgresql-pwd -n shipping
+
+:: Delete Kafka configmap
+kubectl delete configmap kafka-brokers -n shipping
+
```

Comment:
This should get rolled up in deleting the namespace, but since other items here are explicitly deleted before deleting the `shipping` namespace, I figured I'd mention it. There is no deletion of the artifact that gets created in the `install-app.sh` analog of the command `kubectl apply -f $SCRIPTLOC/kafka-topics-configmap.yaml -n shipping`.

Reply:
Good spot!  I'll add the missing delete.  I did intend to be explicit about deleting each resource, rather than just letting the namespace take care of it, partly to document what's happening and also because I can imagine it being useful to understand how to undo / redo parts of the solution.